frontend/src/features/permits/types/application.ts

```diff
@@ -39,6 +40,7 @@ export interface Application extends PartialApplication {
   updatedDateTime?: Nullable<Dayjs>;
   permitData: PermitData;
   applicant?: Nullable<string>;
+  rejectionHistory?: Nullable<ApplicationRejectionHistory[]>;
```

Comment:
You need to consider a lot more things than simply adding it as a field to this type. For example, when creating/updating an application or amending a permit, would we still need to pass this field as part of the request payload data? Or leave it out? Also, would this field need to be a part of ApplicationFormData (which is specifically being used for populating the application/amendment forms, and is indirectly tied to the Application type). It might also be better to not include this in the Application data at all, but instead get it via another mechanism, similar to the transaction history for amendments (where its not a part of the application data, but a separate endpoint entirely)

Reply:
We might need to have a discussion with Praveen regarding this, to see if it's a good idea to include rejectionHistory with the application data